src/app/find-help/reply-offer/reply-offer.component.html

```diff
@@ -61,19 +61,19 @@ <h3>{{ "REPLY_OFFER" | translate }}</h3>
         </mat-form-field>
       </div>
       <div class="row mt-3 mb-3">
-        <mat-checkbox
-          required
-          color="primary"
-          appearance="outline"
-          [(ngModel)]="data.tosApproved"
-          (change)="onConsentChange()"
-          name="consent"
-        >
-          <span *ngIf="!authService.isLoggedIn()">
+        <span *ngIf="!authService.isLoggedIn()">
```

Comment:
I think instead of span ng-container here should work without producing unnecesary element

Reply:
poprawione